server/lib/rippled.js

```diff
@@ -37,16 +42,24 @@ const formatPaychannel = d => ({
   settleDelay: d.SettleDelay,
 });
 
-// generic RPC query
-const query = options => {
+const executeQuery = (url, options) => {
   const params = { ...options, headers: { 'X-User': HOSTNAME } };
-  return axios.post(URL, params).catch(error => {
+  return axios.post(url, params).catch(error => {
     const message = error.response && error.response.data ? error.response.data : error.toString();
     const code = error.response && error.response.status ? error.response.status : 500;
-    throw new utils.Error(`URL: ${URL} - ${message}`, code);
+    throw new utils.Error(`URL: ${url} - ${message}`, code);
   });
 };
 
+// generic RPC query
+function query(...options) {
+  return executeQuery(URL, ...options);
+}
+
+function adminQuery(...options) {
+  return executeQuery(P2P_URL, ...options);
```

Comment:
Why is adminQuery just query `P2P_URL`?

Is the P2P server (whichever P2P refers to going to open an `admin` port? 

Or is this just for gateway balances?

Reply:
I'm guessing (and hoping) the latter.